apps/atomy-q/API/tests/Feature/Api/V1/VendorApiTest.php

```
@@ -0,0 +1,268 @@
+        $this->createVendor($tenantId, ['display_name' => 'Alpha Trading']);
+        $this->createVendor($tenantId, ['display_name' => 'Bravo Supplies']);
+        $this->createVendor((string) Str::ulid(), ['display_name' => 'Other Tenant Vendor']);
+
+        $response = $this->getJson('/api/v1/vendors', $this->authHeaders($tenantId, (string) $user->id));
+
+        $response->assertOk();
+        $response->assertJsonCount(2, 'data');
+        $response->assertJsonPath('data.0.display_name', 'Alpha Trading');
+        $response->assertJsonPath('data.1.display_name', 'Bravo Supplies');
+        $response->assertJsonMissing(['display_name' => 'Other Tenant Vendor']);
+    }
```

PHPMD (2.15.0)

[error] The method test_list_vendors_for_current_tenant is not named in camelCase. (undefined)

(CamelCaseMethodName)